lib/ClangImporter/ImportMacro.cpp

```diff
@@ -509,7 +564,14 @@ static ValueDecl *importMacro(ClangImporter::Implementation &impl,
         }
       }
 
-      // FIXME: If the identifier refers to a declaration, alias it?
+      /* Create an alias for any Decl */
+      clang::Sema &S = impl.getClangSema();
+      clang::LookupResult R(S, {{tok.getIdentifierInfo()}, {}},
+                            clang::Sema::LookupAnyName);
+      if (S.LookupName(R, S.TUScope))
+        if (R.getResultKind() == clang::LookupResult::LookupResultKind::Found)
+          if (const auto *VD = dyn_cast<clang::ValueDecl>(R.getFoundDecl()))
+            return importDeclAlias(impl, DC, VD, name);
```

Comment:
Will this cause trouble where someone does something like
```
int realValue;

#define aliasedName realValue
```
since there's only a _getter_ defined by the code above?

I'm not sure that's a common case, but maybe worth pondering? Should we perhaps restrict it to function declarations for now?

Reply:
I think that I do need it for aliased values as well. Generating a setter for values is totally possible.

Reply:
There, we should now have mutable aliases as well along with appropriate tests to ensure that we do not create a writable alias for a constant value.